sdk/eventhub/azure-eventhubs/azure/eventhub/common.py

```diff
@@ -308,10 +308,10 @@ def try_add(self, event_data):
             raise TypeError('event_data should be type of EventData')
 
         if self._partition_key:
-            if event_data.partition_key and not (event_data.partition_key == self._partition_key):
+            if event_data.partition_key and not event_data.partition_key == self._partition_key:
```

Comment:
We can make it `event_data.partition_key != self._partition_key` like the one you modified in another piece of code.

Reply:
changed